packages/notifications/src/component/Toast.tsx

```diff
@@ -138,40 +138,40 @@ export function Toast({ toast, removeToast, getRef, colors, Button }: Props) {
                 }}
               >
                 {toast.notification.actions.primary &&
-                  toast.notification.actions.primary[0] && (
+                  toast.notification.actions.primary.map(primary => (
```

Comment:
I'm not 100% sure if this works as intended as I was trying to simulate how VSCode does this. I will take a second look at that now, I think they put primary buttons behind a caret or something?

Reply:
Seems to work fine!! Maybe we should make one button primary, instead of 3 primary buttons?